DFC.App.Account/Views/EditYourDetails/Body.cshtml

```diff
@@ -29,370 +29,371 @@
+                        Your marketing preferences
+                    </legend>
+                    <div class="govuk-form-group govuk-checkboxes govuk-checkboxes--small">
+
+
+
+                        <govukCheckbox id="@Model.GetElementId("govukCheckBox", "marketing-opt-out")"
+                                       label="To receive marketing from the National Careers Service’s trusted partners"
+                                       AdditionalClass="checkbox-container-item govuk-checkboxes"
+                                       name="MarketingOptIn"
+                                       checked="@Model.Identity.MarketingPreferences.MarketingOptIn">
+                        </govukCheckbox>
+
+                        <govukCheckbox id="@Model.GetElementId("govukCheckBox", "market-research")"
+                                       label="The National Careers Service can contact me for market research"
+                                       AdditionalClass="checkbox-container-item govuk-checkboxes"
+                                       name="MarketResearchOptIn"
+                                       checked="@Model.Identity.MarketingPreferences.MarketResearchOptIn">
+                        </govukCheckbox>
+                    </div>
+
+                    <div class="govuk-form-group">
+                        <button id="@Model.GetElementId("govukButton", "Save")" class="govuk-button" type="submit" value="Save details" name="saveDetails" data-module="govuk-button">
+                            Save my changes
+                        </button>
+                    </div>
+
+                    <p>or <a href="~/your-account/home">cancel and return to your account</a>.</p>
+
+                </form>
+            </div>
+        </div>
+    </div>
 </div>
+    <script src="https://dev-cdn.nationalcareersservice.org.uk/nationalcareers_toolkit/js/jquerybundle.min.js?jq/NP1aPZeQDFG2sHhiMEg==" type="text/javascript" nonce="yFoYA0sTl5DRrT10/O0u5HWg"></script>
+    <script src="https://dev-cdn.nationalcareersservice.org.uk/nationalcareers_toolkit/js/all.min.js?Uoa/MIvs&#x2B;BY3BaNt9sMilw==" type="text/javascript" nonce="yFoYA0sTl5DRrT10/O0u5HWg"></script>
 
+    <script nonce="yFoYA0sTl5DRrT10/O0u5HWg">window.GOVUKFrontend.initAll()</script>
 
-<script src="https://dev-cdn.nationalcareersservice.org.uk/nationalcareers_toolkit/js/jquerybundle.min.js?jq/NP1aPZeQDFG2sHhiMEg==" type="text/javascript" nonce="yFoYA0sTl5DRrT10/O0u5HWg"></script>
-<script src="https://dev-cdn.nationalcareersservice.org.uk/nationalcareers_toolkit/js/all.min.js?Uoa/MIvs&#x2B;BY3BaNt9sMilw==" type="text/javascript" nonce="yFoYA0sTl5DRrT10/O0u5HWg"></script>
-
-<script nonce="yFoYA0sTl5DRrT10/O0u5HWg">window.GOVUKFrontend.initAll()</script>
+    <script src="https://dev-cdn.nationalcareersservice.org.uk/nationalcareers_toolkit/js/dfcdigital.min.js?zzg1RfbkBBhUqRJBIrwI4A==" nonce="yFoYA0sTl5DRrT10/O0u5HWg"></script>
```

Comment:
Can we use CDN instead of hardcoded JS.

Reply:
Line has been removed as it's being imported through Composite anyway